dev-aws/kafka-shared-msk/account-identity/account.tf

```diff
@@ -143,6 +143,13 @@ module "account_identity_account_api_atomic_projector" {
   cert_common_name = "account-platform/account_api_atomic_projector"
 }
 
+module "account_identity_account_api_change_notifier" {
+  source           = "../../../modules/tls-app"
+  consume_topics   = [kafka_topic.account_identity_account_atomic_v1.name]
+  consume_groups   = ["account-identity.account-api-change-notifier"]
+  cert_common_name = "account-platform/account_api_atomic_projector"
```

Comment:
should this be renamed?

Reply:
The idea was to reuse a certificate if possible so we don´t need completely separate certificates for every single deployment.
In this case both these services are part of the same system and consume the same topic so thought it made sense.

Reply:
Makes sense. Just noticed we did it differently in the verification system - and thought will follow suit.

But I agree, we can keep these the same. Let's keep an 👁️ out for consistency.

Reply:
The cert name should be more generic like "account-api" so it makes to reuse across all account-api subsystems

Reply:
Let's fix it then.